### ISSUE TITLE
Refactor usage of `.kui*` classes in tests

### ISSUE THREAD

**Issue by joshuarrrr** [opensearch-project/OpenSearch-Dashboards#3387]

**Comment by sayuree** [comment]
@joshuarrrr So we need to rename existing class and its occurrences, right? Do we still need to migrate it to  https://github.com/opensearch-project/opensearch-dashboards-functional-test?

**Comment by joshuarrrr** [comment]
> So we need to rename existing class and its occurrences, right? 

Yeah, although it will need some investigation - you may not even need to rename the class if there's another way to reference those elements. Or a test-id may be a better choice.

> Do we still need to migrate it to https://github.com/opensearch-project/opensearch-dashboards-functional-test?

Yes, but let's do that in a separate PR. So the first PR will remove the class and update the existing test, and the second PR will be to migrate/recreate the test in the functional test repository.

**Comment by Aigerim-ai** [comment]
I spotted kui here as well, but I it has its test ID

<img width="909" alt="Screenshot 2023-04-02 at 13 48 45" src="https://user-images.githubusercontent.com/26738268/229350920-b20e1908-f65d-4143-af48-545e7f77d1a9.png">
